react/features/remote-video-menu/components/web/PrivateMessageMenuButton.js

```diff
@@ -59,4 +77,19 @@ class PrivateMessageMenuButton extends Component<Props> {
     }
 }
 
+/**
+ * Maps part of the Redux store to the props of this component.
+ *
+ * @param {Object} state - The Redux state.
+ * @param {Props} ownProps - The own props of the component.
+ * @returns {Props}
+ */
+function _mapStateToProps(state: Object, ownProps: Props): $Shape<Props> {
+    return {
+        ..._abstractMapStateToProps(state, ownProps),
+        _hidden: typeof interfaceConfig !== 'undefined'
+            && (interfaceConfig.DISABLE_PRIVATE_MESSAGES || !(new Set(interfaceConfig.TOOLBAR_BUTTONS)).has('chat'))
```

Comment:
Maybe use Array function instead of converting to set? I guys includes or indexOf will do the job.

Reply:
I reused the logic found in Toolbox.js and Toolbar.js

Reply:
I still don't like it :) In order to save one typeof check we will convert the whole list to set and we will do this on every change in the redux store and we will also do it 3 times (1 here, 1 in Toolbox.js and 1 in Toolbar.js)... This is just bad!

If we want to use it like this the proper solution would be to convert `interfaceConfig.TOOLBAR_BUTTONS` to `Set` only once somewhere during the init of the app and then use it as a `Set`. Practically this conversion will be executed on every redux change now, without a need...

Anyway since you mentioned "reusing", let's reuse `isButtonEnabled` from toolbox/functions. I guess this would be the most appropriate solution in this case. WDYT?

Reply:
@zbettenbuk ^^^